qan-api2/db.go

```diff
@@ -133,8 +133,7 @@ func DropOldPartition(db *sqlx.DB, days uint) {
 	err := db.Select(
 		&partitions,
 		query,
-		days,
-	)
+		days)
```

Comment:
I suggest we get rid of this rule. I think it's easier to navigate the codebase with bracket on new line.

Reply:
If you interested in this change you can create poll for BE devs and if majority will prefer bracket on newline then you can rewrite it in all codebase 😁

Reply:
I agree :+1:

Reply:
We have democracy here, go on)